docs/pubsub-usage.rst

```diff
@@ -20,24 +32,27 @@ Create a new topic for the default project:
 
 .. doctest::
 
-   >>> from gcloud.pubsub import Topic
-   >>> topic = Topic('topic_name')
+   >>> from gcloud import pubsub
+   >>> client = pubsub.Client()
+   >>> topic = client.topic('topic_name')
    >>> topic.create()  # API request
 
-Create a new topic for an explicit project:
+Create a new topic for an explicit client:
 
 .. doctest::
 
-   >>> from gcloud.pubsub import Topic
-   >>> topic = Topic('topic_name', project='my.project')
+   >>> from gcloud import pubsub
+   >>> client = pubsub.Client()
+   >>> topic = Topic('topic_name', client=client)
    >>> topic.create()  # API request
 
-Check for the existance of a topic:
+Check for the existence of a topic:
 
 .. doctest::
 
-   >>> from gcloud.pubsub import Topic
-   >>> topic = Topic('topic_name')
+   >>> from gcloud import pubsub
+   >>> client = pubsub.Client()
+   >>> topic = client.topic('topic_name')
```

Comment:
Same note as above -- would it be useful to have a `get_topic()` method (or similar) that does this API request in one go?